core/src/main/scripts/importer/validateData.py

```diff
@@ -2435,6 +2438,9 @@ def parse_genomic_coord(self, value, column_number):
         Return the parsed value if valid, None otherwise.
         """
         parsed_value = None
+        # convert possible scientific notation to python scientific notation
+        if "e+" in value:
+            value = float(value.replace("e+", "e"))
         try:
             parsed_value = int(value)
         except ValueError:
```

Comment:
The same issue as in segment-wise CNA data files here: please log the appropriate message about the validity of the data file format if this field is not a floating-point number instead of letting the validation script crash with an exception, and also log a message if the floating-point number that does not make sense as a genomic coordinate.